kong/conf_loader/init.lua

```diff
@@ -9,7 +9,7 @@ local openssl_x509 = require "resty.openssl.x509"
 local pl_stringio = require "pl.stringio"
 local pl_stringx = require "pl.stringx"
 local socket_url = require "socket.url"
-local constants = require "kong.constants"
+local constants = require "kong.conf_loader.constants"
```

Comment:
```suggestion
local conf_constants = require "kong.conf_loader.constants"
```

to avoid confusing this one with the `kong/constants.lua` that we typically localize as `constants`

Reply:
done, thanks.